packages/xo-web/src/common/xo/register-proxy-modal/index.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'intl'
+import Component from 'base-component'
+import Icon from 'icon'
+import React from 'react'
+import SingleLineRow from 'single-line-row'
+import { Col, Container } from 'grid'
+import { Input as DebounceInput } from 'debounce-input-decorator'
+
+export default class RegisterProxyModal extends Component {
+  state = {
+    address: undefined,
+    authenticationToken: undefined,
+    name: undefined,
+    vmUuid: undefined,
+  }
+  get value() {
+    return this.state
+  }
+
+  render() {
+    const { address, authenticationToken, name, vmUuid } = this.state
+    return (
+      <Container>
```

Comment:
Don't you think it would be nice to link to our blogpost? https://xen-orchestra.com/blog/xo-proxy-a-concrete-guide/

This would make the point of this feature much clearer IMHO :slightly_smiling_face: